quattor/functions/validation.pan

```diff
@@ -139,3 +139,11 @@ function valid_absolute_file_paths = {
     };
     true;
 };
+
+@documentation{
+    desc = This types is meant to be used to check if strings provided are \
```

Comment:
remove the `desc = `, and you can also get rid of the `\` at the end of each line